.github/workflows/ci.yml

```diff
@@ -42,11 +42,11 @@ jobs:
         - { name: Linux Clang,                    os: ubuntu-22.04, flags: -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -GNinja , gcovr_options: '--gcov-executable="llvm-cov-$CLANG_VERSION gcov"' }
         - { name: Linux GCC DRM,                  os: ubuntu-22.04, flags: -DSFML_USE_DRM=ON -DSFML_RUN_DISPLAY_TESTS=OFF -GNinja }
         - { name: Linux GCC OpenGL ES,            os: ubuntu-22.04, flags: -DSFML_OPENGL_ES=ON -DSFML_RUN_DISPLAY_TESTS=OFF -GNinja }
-        - { name: macOS x64,                      os: macos-13, flags: -GNinja }
-        - { name: macOS x64 Xcode,                os: macos-13, flags: -GXcode }
-        - { name: macOS arm64,                    os: macos-14, flags: -GNinja -DSFML_RUN_AUDIO_DEVICE_TESTS=OFF }
-        - { name: iOS,                            os: macos-14, flags: -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_ARCHITECTURES=arm64 }
-        - { name: iOS Xcode,                      os: macos-14, flags: -DCMAKE_SYSTEM_NAME=iOS -GXcode -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO }
+        - { name: macOS x64,                      os: macos-15, flags: -GNinja }
+        - { name: macOS x64 Xcode,                os: macos-15, flags: -GXcode }
```

Comment:
If we upgrade these jobs to macOS 15 then they will no longer be x64 jobs. Frankly I see little value in explicitly testing on Intel Macs. SFML uses no intrinsics, inline assembly, or prebuilt binaries so I don't see wy architecture differences would matter. Besides, macOS 13 is _old_ and it's safe to assume very few users, if any, are still using SFML 3 on it.

Reply:
At the very least we should target x64 (cross-compiling) if we do decide to update. And buildbot is still running on the Intel mac mini (for now).